src/vs/editor/contrib/inlineCompletions/test/browser/utils.ts

```diff
@@ -131,3 +134,57 @@ export class GhostTextContext extends Disposable {
 		CoreEditingCommands.DeleteLeft.runEditorCommand(null, this.editor, null);
 	}
 }
+
+function generateRandomSimpleString(stringLength: number): string {
+	let randomText: string = '';
+	const possibleCharacters: string = ' abcdefghijklmnopqrstuvwxyz0123456789';
+	for (let i = 0; i < stringLength; i++) {
+		randomText += possibleCharacters.charAt(Math.floor(Math.random() * possibleCharacters.length));
+
+	}
+	return randomText;
+}
+
+export function generateRandomMultilineString(numberOfLines: number, maximumLengthOfLines: number = 20): string {
+	let randomText: string = '';
+	for (let i = 0; i < numberOfLines; i++) {
+		const lengthOfLine = Math.floor(Math.random() * maximumLengthOfLines);
```

Comment:
There is a pattern here that should be refactored into its own function (computing a whole random number between 0 and `maximumLengthOfLines` exclusive).
However, when you use the mersenne twister, there is a helper function for that.